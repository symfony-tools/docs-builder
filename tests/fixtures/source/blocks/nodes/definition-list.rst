
Sed do eiusmod
    Tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.

Quis nostrud
    Exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.

Duis aute irure
    Dolor in reprehenderit in voluptate velit esse.
