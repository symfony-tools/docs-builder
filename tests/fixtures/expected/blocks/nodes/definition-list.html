<dl>
    <dt>Sed do eiusmod</dt>
    <dd> Tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam. </dd>

    <dt>Quis nostrud</dt>
    <dd> Exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </dd>

    <dt>Duis aute irure</dt>
    <dd> Dolor in reprehenderit in voluptate velit esse. </dd>
</dl>
